certbot/certbot/_internal/cli/__init__.py

```diff
@@ -371,6 +371,11 @@ def prepare_and_parse_args(plugins: plugins_disco.PluginsRegistry, args: List[st
         dest="issuance_timeout",
         default=flag_default("issuance_timeout"),
         help=config_help("issuance_timeout"))
+    helpful.add(
+        [None, "certonly", "run"], "--certificate-validity", type=nonnegative_int,
```

Comment:
Fair enough. We can sit on it for a while and gather some other opinions.

Maybe `--certificate-lifetime`? But I am deathly paranoid that the other shoe will drop and some other CA will implement `notBefore` support, which will be an even bigger naming problem.

Reply:
I am still not sure what to do about the name, but I'd like for the name to contain the time unit, in accordance with the convention on (most) of our other similar flags: `--certificate-validity-seconds`.

I am also a little tempted to make it `--certificate-validity-days` and create `non_negative_float` to suit for it, but I'm on the fence with that as well. Having to type out 2592000 comes off as not so great UX to me, but I think it might require a little thought about what to do here.